lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -501,6 +500,12 @@ public function prepare_item_for_response( $user, $request ) {
 			'extra_capabilities' => $user->caps,
 		);
 
+		$schema = $this->get_item_schema();
+
+		if ( ! empty( $schema['properties']['avatar_urls'] ) ) {
+				$data['avatar_urls'] = rest_get_avatar_urls($user->user_email);
```

Comment:
Indentation is a bit off here. Also `rest_get_avatar_urls()` needs spaces around the parameter: `rest_get_avatar_urls( $user->user_email );`